tools/RAiDER/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .era5 import ERA5
+from .era5t import ERA5T
+from .gmao import GMAO
+from .hres import HRES
+from .hrrr import HRRR, HRRRAK
+
+__all__ = ['HRRR', 'HRRRAK', 'GMAO', 'ERA5', 'ERA5T', 'HRES']
```

Comment:
Allows me to import these classes from models instead of having to use each weather model file directly.

Reply:
I thought we made it agnostic for users if its HRRR it can do conus and alaska. Adding it under allowed models might break assumptions from @jlmaurer

Reply:
> Can you combine this with models.allowed.ALLOWED_MODELS?

the `__all__` reserved keyword is for importing from a module. It allos me to write `from RAiDER.models import HRES` rather than `from RAiDER.models.hres import HRES`. It's not changing any functionality.

That said, I believe it is good practice to decouple the constant `ALLOWED_MODELS` and those objects listed in `__all__` (partly because flake8 says so: https://stackoverflow.com/a/49266468). I don't think we want `__all__ = ALLOWED_MODELS + ...`. Also, I am including `HRRRAK` and we could potentially add more constants or functions not all of which are models, so the reverse `ALLOWED_MODELS = RAiDER.models.__all__` would not work.

> I thought we made it agnostic for users if its HRRR it can do conus and alaska. Adding it under allowed models might break assumptions from @jlmaurer

From what I can see, there is an instance method for [`_fetch`](https://github.com/dbekaert/RAiDER/blob/dev/tools/RAiDER/models/hrrr.py#L82-L102) and this does what you are discussing, i.e. fetching the weather model files for `CONUS` or `AK` depending on the bounds. I am using properties of the weather model instances. Unfortunately, this does not impact the instance properties. To update the classes to dynamically update the properties I am using (`_date_range`) could be a lot more work.